contrib/digitalocean/deis-digitalocean.tf

```diff
@@ -0,0 +1,20 @@
+provider "digitalocean" {
+    token = "${var.token}"
+}
+
+resource "digitalocean_droplet" "deis" {
+    count = "${var.instances}"
+    image = "coreos-stable"
+    name = "deis-${count.index+1}"
```

Comment:
previously we had an option to pass a prefix to the script. This feature seems to be gone.

Would it be worth to restore this?